src/content/en/tools/lighthouse/audits/console-time.md

```diff
@@ -0,0 +1,39 @@
+project_path: /web/_project.yaml
+book_path: /web/tools/_book.yaml
+description: Reference documentation for the "Site Does Not Use console.time() In Its Own Scripts" Lighthouse audit.
+
+{# wf_updated_on: 2016-12-01 #}
+{# wf_published_on: 2016-12-01 #}
+
+# Site Does Not Use console.time() In Its Own Scripts  {: .page-title }
+
+## Why the audit is important {: #why }
+
+If you're using `console.time()` to measure your page's performance, consider
+using the User Timing API instead. Benefits include:
+
+* High-resolution timestamps.
+* Exportable timing data.
+* Integration with the Chrome DevTools Timeline. When the User Timing function
+  `performance.measure()` is called during a Timeline recording, DevTools
+  automatically adds the measurement to the Timeline's results.
```

Comment:
would be cool to include a screenshot of the timeline annotated with a user timing. chromestatus.com does some user timings if you want to take it from there.